src/web/app/pages-help/instructor-help-page/instructor-help-page.component.html

```diff
@@ -3,7 +3,7 @@ <h1 class="color-orange"> Help for Instructors</h1>
   <div>
     <p>
       Have questions about how to use TEAMMATES? This page answers some frequently asked questions.
-      <br> If you are new to TEAMMATES, our <a routerLink="../getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
+      <br> If you are new to TEAMMATES, our <a routerLink="/web/front/getting-started">Getting Started</a> guide will introduce you to the basic functions of TEAMMATES.
```

Comment:
Considering that instructor help page is routed under two different links `/web/front/help/instructor` and `/web/instructor/help`, the previous way is the way to go.